src/main/resources/application.properties

```diff
@@ -1 +1,5 @@
-
+spring.datasource.url=jdbc:h2:mem:testdb
+spring.datasource.driver-class-name=org.h2.Driver
+spring.datasource.username=sa
+spring.datasource.password=password
+spring.jpa.database-platform=org.hibernate.dialect.H2Dialect
```

Comment:
HINT: you can enable console and check the data inside this DB:
`spring.h2.console.enabled=true`

See more here: https://www.baeldung.com/spring-boot-h2-database